resources/dicts/patrols/beach/border/leaf-bare.json

```diff
@@ -48,7 +48,7 @@
         "s_c, upon seeing several kits trapped by a Twoleg contraption, immediately jumps into the cold harsh water of the sea. {PRONOUN/s_c/subject/CAP} {VERB/s_c/use/uses} a piece of wood as a makeshift raft and {VERB/s_c/paddle/paddles} the kits back to shore where the rest of the patrol can keep them warm, before racing them back to camp to be cuddled into the pelt of a loving queen."
         ],
         "fail_text": [
-        "To the cat's horror they were too late, the bodies of several kits washing ashore. They mutter a prayer before gently picking up the kits, the patrol swims the little bodies back out into the cold ocean, ensuring that the kits rest within the ocean's embrace."
+        "To the cats' horror, they were too late, the bodies of several kits washing ashore. They mutter a prayer before gently picking up the kits, the patrol swims the little bodies back out into the cold ocean, ensuring that the kits rest within the ocean's embrace."
```

Comment:
missed a couple "they"s

Reply:
oops lemme get on that!

Reply:
wait no i think they're meant to be "theys" bc they're an unknown number of kits?

Reply:
and also i interpreted the "they" in the fail text as the cats (so multiple cats) :P but idm changing if it's meant to be one specific cat!

Reply:
oop no that's a good point!